.prettierrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "trailingComma": "es5",
+  "singleQuote": true,
```

Comment:
your prettier changed all the html attr quotes to single, so i set this here to avoid super noisy diffs where you couldn't see what i actually changed